crypto/x509/v3_ia5.c

```diff
@@ -29,22 +29,21 @@ char *i2s_ASN1_IA5STRING(X509V3_EXT_METHOD *method, ASN1_IA5STRING *ia5)
 {
     char *tmp;
 
-    if (!ia5 || !ia5->length)
+    if (ia5 == NULL || ia5->length == 0)
         return NULL;
-    if ((tmp = OPENSSL_malloc(ia5->length + 1)) == NULL) {
+    if ((tmp = OPENSSL_zalloc(ia5->length + 1)) == NULL) {
```

Comment:
This change is at least useless.
It will `memset` to zero an area who will be filled four line after by `memcpy` call.

Reply:
added zalloc instead of malloc ad removed adding termonated zero

Reply:
You should really treat this like C string functions.  @FdaSilvaYY was saying the zalloc was not good.  Use malloc and terminate it properly.  Just our opinion (neither he nor I are members of the project and don't get a vote).

Reply:
Good point. Please revert these two changes.

Reply:
This change was recommended to me by a project member @beldmit in other the PR that is based on this code.
In this case I don't understand @t8m is joking or not. I think joking because he approved this PR.
In this case I don't understand the status of this PR.
Revert It or you submit it?

Reply:
I tend to agree with @t8m 
Sorry, my recommendation was not good enough.

Reply:
fix commited